gensim/test_unitvec.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
```

Comment:
https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/test/test_matutils.py is more suitable place for this test

Reply:
Created pull request there